cns/service/main.go

```diff
@@ -873,6 +892,18 @@ func main() {
 			if err = httpRemoteRestService.SavePnpIDMacaddressMapping(rootCtx); err != nil {
 				logger.Errorf("Failed to fetch PnpIDMacaddress mapping: %v", err)
 			}
+			// Setting primary macaddress if VF is enabled on the nics for swiftv2
+			var wscli wscliInterface
```

Comment:
don't see this getting assigned any value, wouldn't this cause a nil pointer exception?